octoprint_mrbeam/iobeam/laserhead_handler.py

```diff
@@ -330,21 +330,16 @@ def _load_laser_heads_file(self):
                     data = yaml.safe_load(stream)
 
                 if data:
-                    if "laser_heads" in data:
-                        self._lh_cache = data["laser_heads"]
-
-                    if "last_used_lh_serial" in data:
-                        self._last_used_lh_serial = data["last_used_lh_serial"]
-
-                    if "last_used_lh_model_id" in data:
-                        self._last_used_lh_model_id = data["last_used_lh_model_id"]
-
-                    if "correction_settings" in data:
-                        self._correction_settings = data["correction_settings"]
-            except:
-                self._logger.error(
+                    self._lh_cache = data.get("laser_heads")
+                    self._last_used_lh_serial = data.get("last_used_lh_serial")
+                    self._last_used_lh_model_id = data.get("last_used_lh_model_id")
+                    self._correction_settings = data.get("correction_settings")
```

Comment:
I like this, we just have to be careful because the values might revert back to `None`, which I find OK but is different logic than before

Reply:
I checked, the only thing that can be overwritten is the lh_cache all the others are only set here